plugins/thunderbolt-power/meson.build

```diff
@@ -0,0 +1,19 @@
+cargs = ['-DG_LOG_DOMAIN="FuPluginThunderboltpower"']
```

Comment:
Probably `FuPluginThunderboltPower` for convention here.

Reply:
Fixed